_test/test_data_loaders/test_loader_utilites.m

```diff
@@ -34,7 +34,7 @@ function test_FormatCurrentlyNotSupported(this)
         function test_CorrectHeader(this)
             [result,version] = find_root_nexus_dir(f_name(this,'MAP11014.nxspe'),'NXSPE'); % file name has loose relation to the result
             assertEqual(result,'/11014.spe');
-            assertEqual(version,'1.1');
+            assertEqual(version,1.1);
```

Comment:
the potential for narrow mishits for floats may make it preferable to use the char version

Reply:
From now on I have regular comparison for version beeing bigger than the specific number, so this change comes.  If Matlab works correctly, it should use the same `str2double` transformation while interpreting numbers typed in workspace and converting one, returned by hdf, so I do not expect equality being problem too.